### DISCUSSION TITLE
Is Send Later limited to manage 100 messages in Draft folder??

### DISCUSSION THREAD

**Discussion by enzooo757** [Extended-Thunder/send-later#539] [Q&A] [answered]
Hi,

If I put more than 100 messages (scheduled with Send Later) in the Draft folder, Send Later doesn't works with any extra message. If I put 99 messages, Send Later works inmediately with the next message I schedule.

Is the app limited?

Thx!!

**Comment by Maxsurfer** [comment]
Hello,

it may be so.
I have more than 300 messages in the draft folder.

Send-Later has rarely worked for several months.

But I can't delete all the messages for Send-Later.

That's why I've now created a reminder function that reminds me every 3 hours to check whether the messages have been sent, which of course is quite annoying.

Best regards

**Comment by kkmdmdkk** [comment]
This is precisely what prompted me to install the extension. In Gmail the limit of scheduled emails is 100, and there is no payment option to have more.

I have read many official and unofficial tutorials on "Send Later" and I couldn't find anything about it.

And now I find myself in this forum that the program stops working for a user when he has more than 100 scheduled emails.

At work I have to schedule emails to my clients a year in advance, I already have more than 60 clients and I need to schedule 180 emails, 3 for each client. I don't know what program to use anymore.

If the developer is able to set any limit in this regard, my company is willing to make a suggestive financial donation.

**Reply by jakubstaszak-com** [reply]
Hi, what are your requirements and what is your budget? I could consider a lightweight fork of the Send later addon within a few months.

**Comment by Maxsurfer** [comment]
Hello,

I only have 15-20 scheduled emails in the draft folder.
The rest are about 280 unplanned emails.
The total seems to be the problem.

Best regards

**Comment by jikamens** [comment, marked as answer]
There is indeed a bug which can prevent Send Later from detecting scheduled drafts if you have more than 100 messages in a Drafts folder. This is being tracked in #529. We have a fix for the bug which will be in the next Send Later release. I'm going to lock conversation here but leave the discussion open for reference until we've shipped the fix.
